routes/cube_routes.js

```diff
@@ -12,13 +12,14 @@ var {
   build_id_query,
   get_cube_id,
 } = require('../serverjs/cubefn.js');
-var analytics = require('../serverjs/analytics.js');
-var draftutil = require('../serverjs/draftutil.js');
-var carddb = require('../serverjs/cards.js');
+const analytics = require('../serverjs/analytics.js');
+const draftutil = require('../serverjs/draftutil.js');
+const cardutil = require('../dist/util/Card.js');
```

Comment:
this line is causing tests to fail, is this directory not visible when testing? is it a casing issue?